client/src/components/TripDetails/TripDetails.css

```diff
@@ -1,108 +1,119 @@
-  grid-column: 1 / span 1;
-  grid-row: 3 / span 1;
+.TripDetails__title {
   color: var(--purple);
-  font-size: 2.25rem;
-  margin: 0 3rem;
-}
-.TripDetails__overview-text {
-  grid-column: 2 / span 1;
-  grid-row: 1 / span 1;
-  font-size: 1.125rem;
-  margin-top: 0.2rem;
-}
-.TripDetails__dates-container {
-  grid-column: 2 / span 1;
-  grid-row: 2 / span 1;
-  margin-top: 0.5rem;
+  font-size: 1.75rem;
+  margin: 0;
 }
+.TripDetails__text,
 .TripDetails__dates-text {
   font-size: 1.125rem;
-  margin-top: 0;
-  margin-bottom: 0;
+  margin: 0;
 }
-.TripDetails__location-text {
-  grid-column: 2 / span 1;
-  grid-row: 3 / span 1;
-  font-size: 1.125rem;
-  margin-top: 1rem;
+@media only screen and (min-width: 576px) and (max-width: 767px) {
+  .TripDetails__right {
+    width: 95%;
+  }
+  .TripDetails__container {
+    text-align: start;
+  }
+  .TripDetails__title {
+    font-size: 1.75rem;
+    margin: 1rem 0;
+  }
+  .TripDetails__text {
+    font-size: 1.25rem;
+  }
+  .TripDetails__dates-details {
+    display: flex;
+    flex-direction: row;
+  }
 }
-
-@media only screen and (max-width: 1100px) {
+@media only screen and (min-width: 768px) and (max-width: 991px) {
   .TripDetails {
-    flex-direction: column;
-    align-items: center;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    text-align: start;
+  }
+  .TripDetails__carousel-container {
+    width: 56.25%;
   }
-  .TripDetails__left {
+  .TripDetails__image {
     width: 100%;
+    height: 100%;
   }
   .TripDetails__right {
+    width: 40%;
+  }
+  .TripDetails__container,
+  .TripDetails__dates-container {
     width: 100%;
-    display: flex;
-    flex-direction: column;
   }
-  .TripDetails h2 {
-    font-size: 1.5rem;
-    align-self: center;
+  .TripDetails__title {
+    font-size: 1.75rem;
   }
-  .TripDetails p {
+  .TripDetails__text,
+  .TripDetails__dates-text {
     font-size: 1rem;
-    align-self: flex-start;
-  }
-  .TripDetails .CarouselDot {
-    height: 1rem;
-    width: 1rem;
   }
 }
-
-@media only screen and (max-width: 600px) {
+@media only screen and (min-width: 992px) {
+  .TripDetails {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+  .TripDetails__carousel-container {
+    width: 56.25%;
+  }
   .TripDetails__image {
-    width: 20rem;
-    height: 15rem;
+    width: 100%;
+    height: 100%;
+  }
+  .TripDetails__right {
+    width: 38%;
+  }
+  .TripDetails__container,
+  .TripDetails__dates-container {
+    display: flex;
+    flex-direction: row;
+    text-align: start;
+    width: 100%;
+  }
+  .TripDetails__dates-container,
+  .TripDetails__location-container {
+    margin-top: 3rem;
+  }
+  .TripDetails__location-container {
+    align-items: center;
+  }
+  .TripDetails__title {
+    font-size: 2.25rem;
+    min-width: 40%;
+  }
+  .TripDetails__text {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 60%;
   }
 }
```

Comment:
Change height and width percentages to rem where needed. Check in with the responsive team if you need to!